src/StatusCodeSender.php

```diff
@@ -47,7 +51,10 @@ function send(Request $request) {
             case 422:
                 throw new UnprocessableEntityException("GET request lacked required fields." . $response->getStatusCode());
             case 429:
-                throw new TooManyRequestsException("When using public \"website key\" authentication, we restrict the number of requests coming from a given source over too short of a time." . $response->getStatusCode());
+                $message = json_decode($response->getPayload(), true, 10);
+                $tooManyRequests = new TooManyRequestsException($message['errors'][0]['message'] .  $response->getStatusCode());
```

Comment:
We should probably handle the case where more than one error is returned. Just concatenate the error messages.

Reply:
Good point.